zh/restore-from-aws-s3.md

```diff
@@ -0,0 +1,275 @@
+
+
+## 将指定备份数据恢复到 TiDB 集群
+
++ 创建 `Restore` CR，通过 accessKey 和 secretKey 授权的方式恢复集群：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f resotre-aws-s3.yaml
+    ```
+
+    `restore-aws-s3.yaml` 文件内容如下：
+
+    ```yaml
+    ---
+    apiVersion: pingcap.com/v1alpha1
+    kind: Restore
+    metadata:
+      name: demo2-restore-s3
+      namespace: test2
+    spec:
+      br:
+        cluster: demo2
+        clusterNamespace: test2
+        # enableTLSClient: false
+        # logLevel: info
+        # statusAddr: <status-addr>
+        # concurrency: 4
+        # rateLimit: 0
+        # timeAgo: <time>
+        # checksum: true
+        # sendCredToTikv: true
+      to:
+        host: <tidb-host-ip>
+        port: <tidb-port>
+        user: <tidb-user>
+        secretName: restore-demo2-tidb-secret
+      s3:
+        provider: aws
+        secretName: s3-secret
+        region: us-west-1
+        bucket: my-bucket
+        prefix: my-folder
+    ```
+
++ 创建 `Restore` CR，通过 IAM 绑定 Pod 授权的方式备份集群：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f restore-aws-s3.yaml
+    ```
+
+    `restore-aws-s3.yaml` 文件内容如下：
+
+    ```yaml
+    ---
+    apiVersion: pingcap.com/v1alpha1
+    kind: Restore
+    metadata:
+      name: demo2-restore-s3
+      namespace: test2
+      annotations:
+        iam.amazonaws.com/role: arn:aws:iam::123456789012:role/user
+    spec:
+      br:
+        cluster: demo2
+        sendCredToTikv: false
+        clusterNamespace: test2
+        # enableTLSClient: false
+        # logLevel: info
+        # statusAddr: <status-addr>
+        # concurrency: 4
+        # rateLimit: 0
+        # timeAgo: <time>
+        # checksum: true
+      to:
+        host: <tidb-host-ip>
+        port: <tidb-port>
+        user: <tidb-user>
+        secretName: restore-demo2-tidb-secret
+      s3:
+        provider: aws
+        region: us-west-1
+        bucket: my-bucket
+        prefix: my-folder
+    ```
+
++ 创建 `Restore` CR，通过 IAM 绑定 ServiceAccount 授权的方式备份集群：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f restore-aws-s3.yaml
+    ```
+
+    `restore-aws-s3.yaml` 文件内容如下：
+
+    ```yaml
+    ---
+    apiVersion: pingcap.com/v1alpha1
+    kind: Restore
+    metadata:
+      name: demo2-restore-s3
+      namespace: test2
+    spec:
+      serviceAccount: tidb-backup-manager
+      br:
+        cluster: demo2
+        sendCredToTikv: false
+        clusterNamespace: test2
+        # enableTLSClient: false
+        # logLevel: info
+        # statusAddr: <status-addr>
+        # concurrency: 4
+        # rateLimit: 0
+        # timeAgo: <time>
+        # checksum: true
+      to:
+        host: <tidb-host-ip>
+        port: <tidb-port>
+        user: <tidb-user>
+        secretName: restore-demo2-tidb-secret
+      s3:
+        provider: aws
+        region: us-west-1
+        bucket: my-bucket
+        prefix: my-folder
```

Comment:
@ran-huang Please close the code block with ```.

Reply:
Fixed in https://github.com/pingcap/docs-tidb-operator/pull/99